applicationconfig/client/src/samples/java/ConfigurationSets.java

```diff
@@ -0,0 +1,126 @@
+import com.azure.applicationconfig.ConfigurationClient;
+import com.azure.applicationconfig.ConfigurationClientCredentials;
+import com.azure.applicationconfig.models.ConfigurationSetting;
+import com.azure.applicationconfig.models.SettingSelector;
+import com.azure.common.annotations.Beta;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+/**
+ * Sample demonstrates how to use Azure Application Configuration to switch between "beta" and "production"
+ * configuration sets.
+ */
+public class ConfigurationSets {
+    private static final String CONNECTION_STRING_KEY = "connection-string";
+    private static final String KEY_VAULT_KEY = "key-vault";
+    private static final String BETA = "beta";
+    private static final String PRODUCTION = "production";
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    public static void main(String[] args) throws NoSuchAlgorithmException, InvalidKeyException, IOException {
+        // Retrieve the connection string from the configuration store.
+        // You can get the string from your Azure portal.
+        String connectionString = "Endpoint=https://azconfig-conniey.azconfig.io;Id=0-l1-s0:AEZrHeRv1w3fJ/2ge29R;Secret=2ZIPIs1kudLe7ZzfaD5UZdZotXJysw+Zs/Uc6GbWmqA=";
```

Comment:
This secret shouldn't be public. Once you removed it from here, could you please regenerate the key?

Reply:
Oh junker doodles.